node/status_node_services.go

```diff
@@ -52,6 +52,7 @@ var (
 	// ErrRPCClientUnavailable is returned if an RPC client can't be retrieved.
 	// This is a normal situation when a node is stopped.
 	ErrRPCClientUnavailable = errors.New("JSON-RPC client is unavailable")
+	OpenseaKeyFromEnv string
```

Comment:
Maybe add a comment that the variable is injected at build time, so that's it's more explicit

Reply:
ok

Reply:
@cammellos done